ngraph/test/op_eval/clamp.cpp

```diff
@@ -0,0 +1,403 @@
+//*****************************************************************************
+// Copyright 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "util/engine/interpreter_engine.hpp"
+#include "util/engine/test_engines.hpp"
+#include "util/test_case.hpp"
+#include "util/test_control.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+static string s_manifest = "${MANIFEST}";
+
+namespace
+{
+    template <typename T, test::TestCaseType tct = test::TestCaseType::STATIC>
+    void clamp_test(const element::Type& type,
+                    const PartialShape& dynamic_shape,
+                    const Shape& static_shape,
+                    const std::vector<T>& input,
+                    double min,
+                    double max,
+                    const std::vector<T>& output)
+    {
+        auto data = make_shared<op::Parameter>(type, dynamic_shape);
+        auto clamp = make_shared<op::Clamp>(data, min, max);
+        auto function = make_shared<Function>(clamp, ParameterVector{data});
+
+        auto test_case = test::TestCase<test::INTERPRETER_Engine, tct>(function);
+        test_case.template add_input<T>(static_shape, input);
+        test_case.template add_expected_output<T>(static_shape, output);
+        return test_case.run();
+    }
+}
+#if 0
```

Comment:
Why this code is disabled? If it was needed to debug it should be removed, or commented why we need it and why it's disabled.

Reply:
I missed to enable it. Done